src/sfizz/BeatClock.h

```diff
@@ -134,6 +139,19 @@ class BeatClock {
      * @brief Get the time signature numerator for each frame of the current cycle.
      */
     absl::Span<const int> getRunningBeatsPerBar();
+
+    /**
+     * @brief Get the last BeatPosition
+     *
+     * @return float
+     */
+    float getLastBeatPosition() const;
```

Comment:
The unit is double. Nothing so important, only the initial implementation techniques have given me paranoia about the precision :) (not probably that important after switching to quantization)
As a policy, I picked double for all time keeping, except float in buffers.

Reply:
OK